### PR TITLE
Feature/#53 basic modal and button

### LINKED ISSUE #53: UI Component/#53 - Basic Modal and Button

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #53

### CODE REVIEW

File: src/app/App.ts [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
+      'GO BACK',
+      handleModalClose,
+      'PLAY',
+      handlePlayGame,
+    );
+
+    Render.render('#sf-app', basicModal.modal);
     /*const vieew = new View().renderGameView(
       'AJAJAJ',
       `../../resources/images/avatars/dog.png`,
```
Comment (synowa) [inline src/app/App.ts]: What's that? Do we need that for anything?

File: styles/components/_button.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.button {
+}
```
Comment (synowa) [inline styles/components/_button.scss:2]: Why is it empty?

File: src/app/App.ts (hunk 1 above)
Comment (synowa) [reply src/app/App.ts]: Ok, I see it was deleted in the next PR.

File: styles/_global.scss [hunk 3]
```diff
@@ -3,7 +3,4 @@
   padding: 0;
   margin: 0;
 }
```
Comment (synowa) [inline styles/_global.scss:5]: Do we need that considering we have Normalize.css?

File: styles/components/_modal.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  // Color /
+  background-color: rgba(0, 0, 0, 0.8);
+  &__container {
+    display: flex;
+    width: 80%;
+    // Display & Box Model /
```
Comment (paula0403) [inline styles/components/_modal.scss]: maybe move this comment higher?

**Review by paula0403 — DISMISSED** [review]

File: styles/_global.scss (hunk 3 above)
Comment (vieraboschkova) [reply styles/_global.scss:5]: Yes, normalize doesn't do that. It gives us more control over the design

File: styles/components/_button.scss (hunk 2 above)
Comment (vieraboschkova) [reply styles/components/_button.scss:2]: It is prepared for the future style definitions from the client

File: styles/components/_modal.scss (hunk 4 above)
Comment (vieraboschkova) [reply styles/components/_modal.scss]: Yes, of course!

**Review by Enessetere — DISMISSED** [review]
Looks good to me, well done

**Review by synowa — DISMISSED** [review]
Good job!

File: styles/components/_view.scss [hunk 5]
```diff
@@ -30,10 +30,10 @@
...
 }
 .button {
-  display: block;
-  width: 120px;
-  height: 30px;
-  margin: 0 auto;
+  // display: block;
+  // width: 120px;
+  // height: 30px;
+  // margin: 0 auto;
```
Comment (SebastianBabinski1) [inline styles/components/_view.scss:36]: Why is it in the comment? Is it necessary right now?

File: styles/components/_modal.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  // Color /
+  background-color: rgba(0, 0, 0, 0.8);
+  &__container {
+    // Display & Box Model /
+    display: flex;
+    width: 80%;
+    height: 80%;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 48px;
```
Comment (SebastianBabinski1) [inline styles/components/_modal.scss:23]: Can you use here more flexible units?

File: styles/components/_view.scss (hunk 5 above)
Comment (vieraboschkova) [reply styles/components/_view.scss:36]: These are properties that were set for SPRINT I release, but will have to be removed / rethought

File: styles/components/_modal.scss (hunk 6 above)
Comment (vieraboschkova) [reply styles/components/_modal.scss:23]: I think it should be defined after we get the layout confirmation from the client. Right now these are to show how it behaves.

**Review by paula0403 — APPROVED** [review]

**Review by Enessetere — APPROVED** [review]
Looks good to me, well done :)

**Pull request merged**